.github/workflows/build.yml

```diff
@@ -83,14 +107,14 @@ jobs:
         run: dotnet publish --configuration ${{ matrix.configuration }} --self-contained true -p:PublishTrimmed=true -p:PublishSingleFile=true -p:TrimmerDefaultAction=copyused -p:SuppressTrimAnalysisWarnings=true -r ${{ matrix.rid }} ./src/Bicep.Cli/Bicep.Cli.csproj
 
       - name: Run Bicep E2E Tests
-        if: ${{ matrix.rid != 'linux-musl-x64' }}
+        if: ${{ matrix.rid != 'linux-musl-x64' && matrix.runTests == 'true' }}
```

Comment:
Could we remove the checks for `linux-musl-x64` and just set `runTests` to `false` for this arch?

Reply:
This step and the following one are related. One or the other runs when runTests == true depending on arch. I think I'd rather keep it symmetric for readability.